{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2020-36049"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "socket.io-parser before 3.4.1 allows attackers to cause a denial of service (memory consumption) via a large packet because a concatenation approach is used."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://blog.caller.xyz/socketio-engineio-dos/",
          "refsource": "MISC",
          "tags": [],
          "url": "https://blog.caller.xyz/socketio-engineio-dos/"
        },
        {
          "name": "https://github.com/bcaller/kill-engine-io",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/bcaller/kill-engine-io"
        },
        {
          "name": "https://github.com/socketio/socket.io-parser/commit/dcb942d24db97162ad16a67c2a0cf30875342d55",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/socketio/socket.io-parser/commit/dcb942d24db97162ad16a67c2a0cf30875342d55"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2021-01-08T12:51Z",
  "publishedDate": "2021-01-08T00:15Z"
}